src/main/java/com/palantir/gradle/externalpublish/ExternalPublishRootPlugin.java

```diff
@@ -52,7 +52,7 @@ public final void apply(Project rootProjectVal) {
                     .set(envVars.envVarOrFromTestingProperty("SONATYPE_USERNAME")
                             .getOrNull());
             repo.getPassword()
-                    .set(envVars.envVarOrFromTestingProperty("SONATYPE_USERNAME")
+                    .set(envVars.envVarOrFromTestingProperty("SONATYPE_PASSWORD")
                             .getOrNull());
```

Comment:
Just need to bump down the publishing plugin to 1.12.0 too other this repo will fail to publish https://github.com/palantir/gradle-external-publish-plugin/blob/c94b5f1b3f8fdc496b4272527edac71765732355/build.gradle#L11